numpy/ctypeslib.py

```diff
@@ -327,6 +327,7 @@ def __array_interface__(self):
         ((ct.c_byte, ct.c_short, ct.c_int, ct.c_long, ct.c_longlong), "i"),
         ((ct.c_ubyte, ct.c_ushort, ct.c_uint, ct.c_ulong, ct.c_ulonglong), "u"),
         ((ct.c_float, ct.c_double), "f"),
+        ((ct.c_bool, ), "b"),
```

Comment:
Nitpick: there is an extra space.

Should have a test before we merge it, but while I don't know this stuff well, change looks good.